tests/baselines/reference/comparisonOperatorWithIdenticalPrimitiveType.errors.txt

```diff
@@ -110,6 +116,8 @@ comparisonOperatorWithIdenticalPrimitiveType.ts(43,24): error TS18050: The value
     var rf5 = e != e;
     var rf6 = null != null;
     var rf7 = undefined != undefined;
+                           ~~~~~~~~~
+!!! error TS2870: This binary expression is never nullish. Are you missing parentheses?

```

Comment:
Why would we get this error here? Surely `undefined` is *always* nullish?

Reply:
i mean, when it's not ES5, or not the global scope, it could be any value, but presumably TS is ignoring that possibility :-)